Assets/_PV1/Assets/Geiger System/Scripts/RadiationSource.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.Audio;
+
+namespace _PV1.Assets.Geiger_System.Scripts
+{
+    //IDamagable
+
+    //Player : IDamagable
+    //HP
+    //Item?
+
+    //RadiationSource
+    //MaxDistance
+    //MaxDamage
+    //TickRate
+
+    //Item
+    //Durability
+
+    //moeten dingen gebeuren
+
+    [RequireComponent(typeof(SphereCollider))]
+    public class RadiationSource : MonoBehaviour
+    {
+        [SerializeField] private float m_maxDamage;
+        [SerializeField] private float m_tickRate;
+        [SerializeField] private SphereCollider m_sphereCollider;
+        [SerializeField] private VisualCue m_visualCue;
+        
+        
+        [SerializeField] private AudioSource _audioSource;
+        [SerializeField] private AudioClip[] _geigerCounter;
+        private AudioClip _geigerCounterClip;
```

Comment:
Loop je script nog goed na of het voldoet aan de conventies. Deze  variabelen gebruiken een andere naming conventie dan de rest van je variabelen in het script.
```suggestion
        [SerializeField] private AudioSource m_audioSource;
        [SerializeField] private AudioClip[] m_geigerCounter;
        private AudioClip m_geigerCounterClip;
```